app/controllers/pages_controller.rb

```diff
@@ -29,4 +33,19 @@ def member_pages
     %w[]
   end
 
+  def has_news_feature?
```

Comment:
These won't be necessary after the `ApplicationController` refactor suggestion to move from `private` to `protected`.